microsoftazuresql_connector.py

```diff
@@ -225,25 +191,32 @@ def _handle_test_connectivity(self, param):
     def _handle_list_tables(self, param):
         action_result = self.add_action_result(ActionResult(dict(param)))
         table_schema = param.get('table_schema')
+        dbname = self.database
 
-        query = "SELECT TABLE_NAME, TABLE_SCHEMA, TABLE_CATALOG FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_TYPE = 'BASE TABLE'"
+        query = "SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_TYPE = %s AND TABLE_CATALOG = %s"
 
         if table_schema:
             if phantom.is_fail(self._check_for_valid_schema(action_result, table_schema)):
                 return phantom.APP_ERROR
-            query += " AND TABLE_SCHEMA = " + "'" + table_schema + "'"
-
+            query += "AND TABLE_SCHEMA = %s"
```

Comment:
Has this part of the code been tested? You are missing a space before "AND" which will probably cause your SQL statement to fail if there is a table schema.

Reply:
Yes i tested this cases and it is passing.
though I updated this query as it will look valid query.